profile.html

```diff
@@ -0,0 +1,164 @@
+            Mauris porttitor pretium elementum. Duis blandit elit tincidunt ipsum ultricies, ut faucibus lorem
+            facilisis. Proin ipsum turpis, pharetra in lorem ac, porta ullamcorper velit. Proin gravida odio eget elit
+            ultricies sodales. Vivamus vel tincidunt ligula. Proin pulvinar pellentesque velit eget luctus. Aliquam
+            vitae enim ut purus vestibulum sollicitudin sit amet eget lacus. Nunc tempus fringilla tincidunt.
+          </p>
+        </div>
+        <hr />
+      </div>
+      <h2 class="text-center mb-3">Friends</h2>
+      <div class="container text-center">
+        <div class="row">
+          <div class="col-sm-3">
+            <div class="card h-100">
+              <img
+                src="/images/pexels-david-garrison-2128807.jpg"
+                class="card-img-top"
+                alt="Man with grey background"
+              />
+              <div class="card-body">
+                <h5 class="card-title">David Garrison</h5>
+                <a href="#" class="btn btn-primary">Profile</a>
+              </div>
+            </div>
+          </div>
+          <div class="col-sm-3">
+            <div class="card h-100">
+              <img src="/images/pexels-justin-shaifer-1222271.jpg" class="card-img-top" alt="Man in green shirt" />
+              <div class="card-body">
+                <h5 class="card-title">Justin Shaifer</h5>
+                <a href="#" class="btn btn-primary">Profile</a>
+              </div>
+            </div>
+          </div>
+          <div class="col-sm-3">
+            <div class="card h-100">
+              <img src="/images/pexels-spencer-selover-428364.jpg" class="card-img-top" alt="Man in profile" />
+              <div class="card-body">
+                <h5 class="card-title">Spencer Selover</h5>
+                <a href="#" class="btn btn-primary">Profile</a>
+              </div>
+            </div>
+          </div>
+          <div class="col-sm-3">
+            <div class="card h-100">
+              <img
+                src="/images/pexels-simon-robben-614810.jpg"
+                class="card-img-top"
+                alt="Man with light grey background"
+              />
+              <div class="card-body">
+                <h5 class="card-title">Simon Robben</h5>
+                <a href="#" class="btn btn-primary">Profile</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+    <hr />
+    <footer class="my-5 pt-5 text-muted text-center text-small container">
+      <p class="mb-1">© 2022 BeSocial</p>
+      <ul class="list-inline">
+        <li class="list-inline-item"><a href="#">Privacy</a></li>
+        <li class="list-inline-item"><a href="#">Terms</a></li>
+        <li class="list-inline-item"><a href="#">Support</a></li>
+      </ul>
+    </footer>
+    <script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
```

Comment:
```suggestion
```

Reply:
When I do this the dropdown to write posts don't work anymore.